data/provider/src/main/java/tech/pegasys/artemis/api/schema/IndexedAttestation.java

```diff
@@ -29,4 +30,12 @@ public IndexedAttestation(
     this.data = new AttestationData(indexedAttestation.getData());
     this.signature = new BLSSignature(indexedAttestation.getSignature());
   }
+
+  public tech.pegasys.artemis.datastructures.operations.IndexedAttestation
+      asInternalIndexedAttestation() {
+    return new tech.pegasys.artemis.datastructures.operations.IndexedAttestation(
+        SSZList.createMutable(attesting_indices, attesting_indices.size(), UnsignedLong.class),
```

Comment:
we could potentially use the defined constants when creating a SSZList since at this point we can identify which one is appropriate
```Constants.MAX_VALIDATORS_PER_COMMITTEE``` appears to be the best max size in this case. not sure if @ajsutton has a preference re. using the current size, or the defined constants.

Reply:
The only function that max size provides is to specify an upper bound on the size of list you have to read in - thus preventing DOS attacks where you get sent stupidly large data.

So the ideal case is to use the correct max size per the spec and enforce that when reading the data.  We'll need to do that kind of protection if we ever intend to be able to make these APIs available publicly.

But otherwise it doesn't really matter.  It's probably safer to use the spec defined max size whenever possible in case some function downstream needs to add to the list (e.g. if we deserialise a state and then process slots it might add a new validator as part of epoch processing).  If you use current size in that case the operation will fail because the max size will be exceeded.